tests/Integration/Behaviour/Features/Context/Domain/Product/CommonProductFeatureContext.php

```diff
@@ -374,7 +374,7 @@ public function assertIsNotIndexed(string $productReference): void
     }
 
     /**
-     * @Then product :productReference should not be indexed for shops ":shopReferences"
+     * @Then product ":productReference" should not be indexed for shops ":shopReferences"
```

Comment:
I'm not sure the double quotes are necessary here? Usually the syntax `:variable` is compatible with or without the double quotes in the feature files

Reply:
I like having double quotes, it makes it more consistent. Because when we use some advanced regex we always use double quotes for references